elasticsearch-vmss/azuredeploy.json

```diff
@@ -0,0 +1,504 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "clusterName": {
+      "type": "string",
+      "defaultValue": "es-azure",
+      "metadata": {
+        "description": "The name of the cluster."
+      }
+    },
+    "vmSku": {
```

Comment:
I disagree, but I made the changes. Having a default value allows ease of deployment and lessens chance of failure, but having the parameter still allows the customer to try different things easily, before production.